Automattic/jetpack
# My Jetpack: Update product icon opacity in Product cards

Updates opacity of product icons to 40% on the product card section.

Fixes #23511

#### Changes proposed in this Pull Request:

* Updates IconWrapper to accept a property opacity defaulting to 1.
* Wrapps every icon on a G element using the opacity argument
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/746152/159327686-f69dfc68-8aba-4d5b-9006-8dee31a42456.png">

#### Before
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/746152/159327733-8b2926fd-bbd5-4ddf-b2b8-300f8ad07058.png">

#### Jetpack product discussion
update/my-jetpack-product-card-button-behaviour

#### Does this pull request change what data or activity we track or use?
no
#### Testing instructions:
1. Checkout this branch, on a connected site
2. Visit My Jetpack
3. Confirm the icons look a 40% opaque.
